lib/fake_braintree/merchant_account.rb

```diff
@@ -0,0 +1,98 @@
+module FakeBraintree
+  class MerchantAccount
+    include Helpers
+
+    def initialize(merchant_account_hash_from_params, options)
+      @merchant_account_hash = {
+        'id' => options[:id]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.